### ISSUE TITLE
Inverted check for HA cluster

### ISSUE THREAD

**Issue by julianbrost** [Icinga/icinga2#9283]
I mean the following just looks like it's the wrong way around:

https://github.com/Icinga/icinga2/blob/bf5b905707502c0424e7f9ed5f7b5fd53dc75dc1/lib/remote/apilistener.cpp#L1809-L1817

Maybe fix it, maybe have a look if code that uses it is even needed as the check was wrong all the time.

**Comment by julianbrost** [comment]
Wait a second... that could actually explain #9265 as this function is used in a check that would delay notifications on failover: https://github.com/Icinga/icinga2/blob/bf5b905707502c0424e7f9ed5f7b5fd53dc75dc1/lib/icinga/notification.cpp#L133-L134

**Comment by yhabteab** [comment]
> Wait a second... that could actually explain [#9265](https://github.com/Icinga/icinga2/issues/9265) as this function is used in a check that would delay notifications on failover:

Not exactly! I mean, yes, that check is broken but even after fixing that check I still get duplicate notifications. That's because if a node receives a `event::SendNotifications` message, there's simply no other event that can make this obsolete. Apart from that, the `next_notification` timestamp seems to be only relevant for the `NotificationComponent` timer and not for the `OnNotificationsRequested` handler.
https://github.com/Icinga/icinga2/blob/520aed6049c6a6142fbe322bfc9b20214bae7290/lib/notification/notificationcomponent.cpp#L220-L221

**Comment by julianbrost** [comment]
Well, that also was just a wild guess from my side that this could be the cause. Can you please add your findings regarding the other issue over there as that's where they are relevant then.
